src/Pdp/PublicSuffixListManager.php

```diff
@@ -20,6 +20,13 @@ class PublicSuffixListManager
 {
     const PDP_PSL_TEXT_FILE = 'public-suffix-list.txt';
     const PDP_PSL_PHP_FILE = 'public-suffix-list.php';
+
+    const ICANN_SECTION = 'ICANN';
```

Comment:
I'd like to call these either `ICANN` and `PRIVATE` or `ICANN_DOMAINS` and `PRIVATE_DOMAINS`. 

Additionally I'd like to have another constant, either `ALL` or `ALL_DOMAINS`, to represent the entire list.